app/views/users/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> WELCOME TO WIKI PAGE</h1>
+<%= form_with model :user do |form|%>
+<%form.label :username, "Username: "%>
+<%= form.text_field :username, placeholder="Enter your username"%><br/>
+<%= form.label  :email, "Email"%>
+<%= form.text_field :email , placeholder="Enter your email address"%><br/>
+<%= form.label  :password, "Password"%>
+<%= form.password_field :password , placeholder="enter your password"%>
+<%= form.submit "Submit"%>
+
+<% end %>
```

Comment:
- [ ] Kindly add a commented-out user form. This user form should be created in regular HTML. This is important since it is a requirement in this project. ⬇️ 
![user_form1](https://user-images.githubusercontent.com/32361963/110918587-63237580-832c-11eb-9387-e3ed06e9a48e.png)

![user_form2](https://user-images.githubusercontent.com/32361963/110918679-7cc4bd00-832c-11eb-8c8a-23b2a78730c2.png)